zomes/rea_economic_resource/lib/src/lib.rs

```diff
@@ -128,8 +127,9 @@ impl API for EconomicResourceZomePermissableDefault {
         // if the event is a transfer-like event, run the receiver's update first
         if let MaybeUndefined::Some(receiver_inventory) = &event.to_resource_inventoried_as {
             let inv_entry_hash: &EntryHash = receiver_inventory.as_ref();
+
             let (meta, resource_address, new_resource, prev_resource) = handle_update_inventory_resource(
-                &get_latest_action_hash(inv_entry_hash.clone())?,   // :TODO: temporal reduction here! Should error on mismatch and return latest valid ID
```

Comment:
I am wondering about the removal of the TODO, and TBH trying to understand it. 
Do we know if that issue is legit resolved now?